builder/dockerfile/evaluator_unix.go

```diff
@@ -4,6 +4,6 @@ package dockerfile
 
 // platformSupports is a short-term function to give users a quality error
 // message if a Dockerfile uses a command not supported on the platform.
-func platformSupports(command string) error {
+func platformSupports(command interface{}) error {
```

Comment:
Yes and actually, with lcow support, we should change this behavior to be "target platform" dependent instead of statically defined